app/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
newline at eof. Is there some sort of content-agnostic Linter we can use to check this?